--- mkvtoolnix-1.5.5/src/mmg/mmg.cpp.orig	2005-08-21 15:04:48.000000000 +0200
+++ mkvtoolnix-1.5.5/src/mmg/mmg.cpp	2005-08-24 22:54:38.000000000 +0200
@@ -2002,6 +2002,7 @@
   cfg = new wxConfig(wxT("mkvmergeGUI"));
   wxConfigBase::Set(cfg);
   cfg->SetPath(wxT("/GUI"));
+  cfg->Write(wxT("installation_path"), wxT("$INSTDIR"));
   cfg->Read(wxT("last_directory"), &last_open_dir, wxT(""));
   for (i = 0; i < 4; i++) {
     k.Printf(wxT("last_settings %u"), i);
