--- mkvtoolnix-7.9.0/ac/ax_boost_base.m4~	2015-05-10 11:04:26.000000000 +0200
+++ mkvtoolnix-7.9.0/ac/ax_boost_base.m4	2015-06-18 21:08:07.846272020 +0200
@@ -94,7 +94,7 @@
       *)    libsubdirs="lib/${host_cpu}-${host_os}" ;;
     esac
     if test $host_cpu = x86_64 -o $host_cpu = ppc64 -o $host_cpu = s390x -o $host_cpu = sparc64; then
-        libsubdirs="$libsubdirs lib64"
+        libsubdirs="$libsubdirs lib64 libx32"
     fi
     libsubdirs="$libsubdirs lib"
 
